client/src/utils/handled-error.ts

```diff
@@ -1,6 +1,6 @@
 import { has } from 'lodash-es';
 
-interface ErrorData {
+export interface ErrorData {
```

Comment:
```suggestion
interface ErrorData {
```
This does not need to be imported.